src/serialize.h

```diff
@@ -189,7 +191,8 @@ enum
     SER_GETHASH         = (1 << 2),
 };
 
-#define READWRITE(obj)      (::SerReadWrite(s, (obj), nType, nVersion, ser_action))
+#define READWRITE(obj)                      (::SerReadWrite(s, (obj), nType, nVersion, ser_action))
+#define READWRITE_POLYMORPHIC(obj, b, d)    (::SerReadWrite<decltype(s), b, decltype(obj), d>(s, (obj), nType, nVersion, ser_action))
```

Comment:
Personally I'd prefer making parameters `b` and `d` names more self-explaining; I guess they stands for `base` and `derived` (correct?), but why not putting some more clear names?

Reply:
+1

Reply:
Makes sense, I have renamed `b, d` to `base, derived`.